src/test/resources/features/List.feature

```diff
@@ -1,6 +1,6 @@
 Feature: List
 
-  @logout
+  @logout @deleteList
   Scenario: Create new List
     Given The user goes to "login" page
     When The "user" fills the form with email and password
```

Comment:
Shouldn't this line be an And?

Reply:
Also, regarding the line that is after..., I think a "s" is missing at the end of the word: create

Reply:
Hi Ale, I already reviewed your comments and solved it.

Reply:
Thanks. Keep up the good work!